R/r.R

```diff
@@ -66,6 +66,82 @@ use_test <- function(name = NULL, open = NULL) {
 }
 
 
+#' Automatically rename paired `R/` and `test/` files
+#'
+#' @description
+#' * Moves `R/{old}.R` to `R/{new}.R`
+#' * Moves `tests/testthat/test-{old}.R` to `tests/testthat/test-{new}.R`
+#' * Moves `tests/testthat/test-{old}-*.*` to `tests/testthat/test-{new}-*.*`
+#'   and updates paths in the test file.
```

Comment:
Is there a conceptual description of these files? Does it have something to do with how you use `verify_output()`? I'm not immediately sure what we're talking about.

Reply:
Me neither.